common/ferrostar/src/navigation_controller/snapshots/ferrostar__navigation_controller__tests__full_route_state_snapshot-2.snap

```diff
@@ -0,0 +1,14132 @@
+          - lat: 45.372855
+            lng: -122.586367
+          - lat: 45.372463
+            lng: -122.586234
+          - lat: 45.372156
+            lng: -122.586125
+          - lat: 45.371846
+            lng: -122.586016
+          - lat: 45.371508
+            lng: -122.585901
+        distance: 513.266
+        duration: 25.269
+        road_name: Cascade Highway
+        instruction: Keep right to take OR 213 South/Cascade Highway.
+        visual_instructions: []
+        spoken_instructions: []
+        annotations: ~
+        incidents: []
+      - geometry:
+          - lat: 45.371508
+            lng: -122.585901
+          - lat: 45.371508
+            lng: -122.585901
+        distance: 0
+        duration: 0
+        road_name: Cascade Highway
+        instruction: You have arrived at your destination.
+        visual_instructions: []
+        spoken_instructions: []
+        annotations: ~
+        incidents: []
+    remaining_waypoints:
+      - coordinate:
+          lat: 45.371508
+          lng: -122.585901
+        kind: Break
+    progress:
+      distanceToNextManeuver: 21.560594461991684
+      distanceRemaining: 534.8265944619916
+      durationRemaining: 26.16489751496642
+    deviation: NoDeviation
+    visual_instruction: ~
+    spoken_instruction: ~
+    annotation_json: ~
+- Navigating:
+    current_step_geometry_index: 19
+    snapped_user_location:
+      coordinates:
+        lat: 45.375657
+        lng: -122.58831
+      horizontal_accuracy: 0
+      course_over_ground:
+        degrees: 72
+        accuracy: 5
+      speed: ~
+    remaining_steps:
+      - geometry:
+          - lat: 45.375411
+            lng: -122.586549
+          - lat: 45.374846
+            lng: -122.587677
+          - lat: 45.374808
+            lng: -122.587796
+          - lat: 45.374781
+            lng: -122.587928
+          - lat: 45.374774
+            lng: -122.588028
+          - lat: 45.37478
+            lng: -122.588136
+          - lat: 45.374789
+            lng: -122.588209
+          - lat: 45.374812
+            lng: -122.588294
+          - lat: 45.374849
+            lng: -122.588384
+          - lat: 45.37488
+            lng: -122.588451
+          - lat: 45.374919
+            lng: -122.588512
+          - lat: 45.374978
+            lng: -122.588569
+          - lat: 45.375024
+            lng: -122.58861
+          - lat: 45.375075
+            lng: -122.58864
+          - lat: 45.375143
+            lng: -122.588661
+          - lat: 45.375324
+            lng: -122.588722
+          - lat: 45.375415
+            lng: -122.588676
+          - lat: 45.375483
+            lng: -122.588617
+          - lat: 45.375559
+            lng: -122.58852
+          - lat: 45.37561
+            lng: -122.588432
+          - lat: 45.375657
+            lng: -122.58831
+          - lat: 45.375703
+            lng: -122.588114
+        distance: 295
+        duration: 12.258
+        road_name: ""
+        instruct
```

Comment:
Apologies in advance for the mega-diff that will crash your browser, but this is the part that doesn't look quite right. I think this should be empyt at the end of the route, but it's not.

Reply:
EDIT: This moved to another file.